pom.xml

```diff
@@ -386,4 +386,16 @@
             </build>
         </profile>
     </profiles>
+    <distributionManagement>
+        <snapshotRepository>
+            <id>snapshots</id>
+            <name>oss-jfrog-artifactory-snapshots</name>
```

Comment:
Agreed offline to use a name based on the project, not on the Artifactory repository.